README.md

```diff
@@ -180,6 +180,58 @@ For more examples, see:
 
 Contributions to `pedalboard` are welcomed! See [CONTRIBUTING.md](https://github.com/spotify/pedalboard/blob/master/CONTRIBUTING.md) for details.
 
+## Frequently Asked Questions
+
+
+### Can Pedalboard be used with live (real-time) audio?
+
+Technically, yes, Pedalboard could be used with live audio input/output. See [@stefanobazzi](https://github.com/stefanobazzi)'s [guitarboard](https://github.com/stefanobazzi/guitarboard) project for an example that uses the `python-sounddevice` library to wire Pedalboard up to live audio.
+
+However, there are a couple big caveats when talking about using Pedalboard in a live context. Python, as a language, is [garbage-collected](https://devguide.python.org/garbage_collector/), meaning that your code randomly pauses on a regular interval to clean up unused objects. In most programs, this is not an issue at all. However, for live audio, garbage collection can result in random pops, clicks, or audio drop-outs that are very difficult to prevent.
```

Comment:
You could pause the garbage collector though?

Another caveat that may be worth mentioning is that the [overhead](https://pythonspeed.com/articles/python-extension-performance/) from serialization/deserialization to/from the C++-bindings.